pkg/controller/v1beta1/inferenceservice/reconcilers/deployment/deployment_reconciler.go

```diff
@@ -216,11 +216,11 @@ func (r *DeploymentReconciler) checkDeploymentExist(ctx context.Context, client
 	}
 	// existed, check equivalence
 	// for HPA scaling, we should ignore Replicas of Deployment
-	// for external scaler, we should not ignore Replicas.
+	// for external or none scaler, we should not ignore Replicas.
```

Comment:
this is the same

Reply:
Same for my question in: https://github.com/kserve/kserve/pull/4363#discussion_r2055621196